leonardo_mejia/task/tic_tac_toe/src/Board.java

```diff
@@ -0,0 +1,27 @@
+public class Board {
```

Comment:
Please document your java code. You would review this page https://www.spiria.com/en/blog/method-and-best-practices/how-document-java-code/